py-polars/polars/dataframe/frame.py

```diff
@@ -3890,11 +3890,15 @@ def write_database(
             Select the engine to use for writing frame data; only necessary when
             supplying a URI string (defaults to 'sqlalchemy' if unset)
         engine_options
-            Additional options to pass to the engine's associated insert method:
-
-            * "sqlalchemy" - currently inserts using Pandas' `to_sql` method, though
-              this will eventually be phased out in favor of a native solution.
-            * "adbc" - inserts using the ADBC cursor's `adbc_ingest` method.
+            Additional options to pass to the insert method associated with the engine
+            specified by the option `engine`.
+            See the documentation for the appropriate method:
```

Comment:
I'd omit this line - the reader is already *in* the documentation (not the documentation of the adbc/alchemy projects, but this suggestion doesn't link to those). But good improvement otherwise :)